### ISSUE TITLE
Add Accessibility label to Strings.swift

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#28119]
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-12899)

**Comment by data-sync-user** [comment]
➤ Cyndi Chin commented:

Here are some QA tickets with some updates to a11y labels:

  * https://mozilla-hub.atlassian.net/browse/FXIOS-13104 ( https://mozilla-hub.atlassian.net/browse/FXIOS-13104|smart-link )  
  * https://mozilla-hub.atlassian.net/browse/FXIOS-13106 ( https://mozilla-hub.atlassian.net/browse/FXIOS-13106|smart-link ) 
  * https://mozilla-hub.atlassian.net/browse/FXIOS-13107 ( https://mozilla-hub.atlassian.net/browse/FXIOS-13107|smart-link ) 

We introduced the strings in this PR - https://github.com/mozilla-mobile/firefox-ios/pull/27947/commits/a1878aef89cfc5eb9d2379edd122c3dbed7d1e7a#r2221582962 ( https://github.com/mozilla-mobile/firefox-ios/pull/27947/commits/a1878aef89cfc5eb9d2379edd122c3dbed7d1e7a#r2221582962|smart-link ) without keys to avoid exporting the strings for translation because they were not final. While I’ll make a change to update the labels, designs should be updated with the final content description and we should confirm the labels with a11y team. So I’ll continue to leave the keys empty to avoid the export. Not for MVP, but we need to keep this in mind if we do want translated strings.

cc: Andres Furlan Issam Mani

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v144.1 (62736) with iPhone 13 Pro (iOS 26).

Please note that I’ve checked the PR from the description, and it relates to the animations, which now look very good.
Additionally, the three tickets mentioned below have been verified.

[^ScreenRecording_10-10-2025 13-35-07_1.MP4]
